src/frontend/scss/components/templates/courses/cms/_program_detail.scss

```diff
@@ -54,6 +55,79 @@
     padding-right: $grid-gutter-width;
   }
 
+  &__objectives {
+    @if r-theme-val(program-detail, checkmark-list-decoration) {
```

Comment:
This statement is useless, as you defined `checkmark-list-decoration` into `_theme.scss`, it will be always defined.